databrowser/src/components/download/DownloadSection.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="mx-auto max-w-5xl">
+    <div class="hidden lg:flex items-stretch space-x-4 h-full">
+      <span class="self-center">Retrieve data based on table settings</span>
+
+      <DownloadButtonCell
```

Comment:
I would make a component here with another name like `DatasetDetails`. And inside this component you could make use of general components like `DownloadCell`, `DownloadTitle` and `DownloadButton`. In addition, like this you have to advantage that you can group logic into the component. For example the `downloadJson` function in that you use in the component below.